### ISSUE TITLE
Release version is not offered as update or on WowUp website

### ISSUE THREAD

**Issue by cremor** [WowUp/WowUp.CF#24]
**Describe the bug**
The first non-beta version of WowUp with CurseForge released yesterday. But it is not offered as an update in the app (currently running beta 11).
Also, https://wowup.io still only offers beta 11.

**To Reproduce**
1. Go to https://wowup.io
2. Check the offered version of WowUp with CurseForge
3. Install beta 11
4. Check the offered version of the in-app updater

**Expected behavior**
Both the website and the in-app updater should offer the release version.

**Comment by jliddev** [comment]
The website should be updated.

You may need to go to options > Application and set the channel to stable.

![image](https://user-images.githubusercontent.com/20467484/213924266-16ec23a6-3e59-429b-9e64-7c01cbfee322.png)

**Comment by cremor** [comment]
Yeah, that was set to beta. Changing it to stable offered the update. But shouldn't it update stable releases anyway if they are newer than all available beta version?

Also, I think that I've never set this option manually. Maybe installing a beta sets it automatically? In that case all existing WowUp with CurseForge users will have it set to beta.
